views/layout.html

```diff
@@ -0,0 +1,124 @@
+{#- We can't mount GOV.UK Frontend's assets at root as it's done automatically by the extensions framework. -#}
+{%- set assetPath = '/govuk/assets' -%}
+
+{% extends "govuk/template.njk" %}
+
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
+{% from "govuk/components/back-link/macro.njk"        import govukBackLink %}
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/character-count/macro.njk"  import govukCharacterCount %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/date-input/macro.njk"       import govukDateInput %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/error-message/macro.njk"    import govukErrorMessage %}
+{% from "govuk/components/error-summary/macro.njk"    import govukErrorSummary %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/file-upload/macro.njk"      import govukFileUpload %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/inset-text/macro.njk"       import govukInsetText %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/phase-banner/macro.njk"     import govukPhaseBanner %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/skip-link/macro.njk"        import govukSkipLink %}
+{% from "govuk/components/summary-list/macro.njk"     import govukSummaryList %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/tabs/macro.njk" import govukTabs %}
+{% from "govuk/components/tag/macro.njk" import govukTag %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/warning-text/macro.njk"     import govukWarningText %}
+
+{% block head %}
+  <!--[if !IE 8]><!-->
+  <link href="{{CDN_URL}}/stylesheets/govuk-frontend/v3.5.0/govuk-frontend-3.5.0.min.css" rel="stylesheet" />
+
+  <!-- TODO BI-3878 This CSS must still be added to CDN -->
+  <link href="{{CDN_URL}}/stylesheets/services/strike-off-objections/application.css" rel="stylesheet" />
+  <!--<![endif]-->
+
+  {# For Internet Explorer 8, you need to compile specific stylesheet #}
+  {# see https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/supporting-internet-explorer-8.md #}
+  <!--[if IE 8]>
+  <link href="{{CDN_URL}}/stylesheets/govuk-frontend/v3.5.0/govuk-frontend-ie8-3.5.0.min.css" rel="stylesheet" />
+
+  <!-- TODO BI-3878 This CSS must still be added to CDN -->
+  <link href="{{CDN_URL}}/stylesheets/services/strike-off-objections/application.css" rel="stylesheet" />
```

Comment:
This line appears to match the line on 37, can we just have one instance outside the browser if statements? Would that still work?

Reply:
Not sure. We had the same with PTF and it might be needed for special handling in different browsers. I'll check with @SVarallo though.

Reply:
I don't see why it wouldn't work to have the if just contain the govuk import and have the application styles outside it.

Reply:
But @ICoxCH 's suggestion is to have the govuk import outside of the if statement. But doing this would mean it would also apply for older browsers (see block below - [if lt IE 9]) when the import shouldn't be done.

My opinion right now, given as developers we're not actually testing these changes with different browsers and I can't be sure of what the change might do, is that we leave this as is, exactly the same as we had for PTF.

Reply:
for IE8 there should be an application-ie8.css.
See govuk-frontend/govuk/all-ie8.scss
Also see the application-ie8.scss in extensions-web

Reply:
I don't think that's required as govuk-frontend-ie8-3.5.0.min.css covers that. In ptf an application-ie8.css wasn't required.

Reply:
in which case we should use govuk-frontend-ie8-3.6.0.min.css as we've moved to 3.6.0 now

Reply:
Done. Now using the v3.6.0 css and page rendering correctly.